portal/server/api.js

```diff
@@ -797,6 +797,24 @@ const getUkefDecisionReport = async (token, payload) => {
   }
 };
 
+const getPreviousUtilisationReportsByBank = async (token, bankId) => {
+  try {
+    const response = await axios({
+      method: 'get',
+      url: `${PORTAL_API_URL}/v1/previous-reports/${bankId}`,
```

Comment:
Same here for some extra validation on the bankID to check it's the right format